routers/home.go

```diff
@@ -142,6 +142,7 @@ func RenderRepoSearch(ctx *context.Context, opts *RepoSearchOptions) {
 		Keyword:            keyword,
 		OwnerID:            opts.OwnerID,
 		AllPublic:          true,
+		AllLimited:         true,
```

Comment:
Shouldn't this be true only if user is authorized?

Reply:
yes - and in this functionroutine it is :) @lafriks